src/components/structures/InteractiveAuth.js

```diff
@@ -61,11 +79,19 @@ export default React.createClass({
             authData: this.props.authData,
             doRequest: this._requestCallback,
             startAuthStage: this._startAuthStage,
+            inputs: this.props.inputs,
+            stateUpdated: this._authStateUpdated,
+            makeRegistrationUrl: this.props.makeRegistrationUrl,
+            matrixClient: this.props.matrixClient,
+            sessionId: this.props.sessionId,
+            clientSecret: this.props.clientSecret,
+            emailSid: this.props.emailSid,
         });
 
         this._authLogic.attemptAuth().then((result) => {
             this.props.onFinished(true, result);
         }).catch((error) => {
+            this.props.onFinished(false, error);
```

Comment:
Yes, I think so. It might also be worth renaming the `onFinished` property here to distinguish it from the one used in dialogs.